### PR TITLE
Background Image issue & New Smart Wallet PFP Image 

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/ProfilePage/OpenSeaNewPFP.tsx [hunk 1]
```diff
@@ -2,49 +2,59 @@
...
-          flex-col flex justify-center items-center
-          gap-y-[20px] md:gap-y-[30px]
-          md:w-[620px] md:h-[320px]
-          bg-white"
+const OpenSeaNewPFP: FC<OpenSeaNewPFPProps> = ({ isModalVisible, toggleIsVisible }) => {
+  const { setIsViewAll, setShouldSelectNewPFP } = useWalletCollectionProvider()
+
+  return (
+    <Modal
+      id="help_snart_wallet"
```
Comment (sweetmantech) [inline components/ProfilePage/OpenSeaNewPFP.tsx]:
```suggestion
      id="help_smart_wallet"
```

File: components/ProfilePage/ProfileToken.tsx [hunk 2]
```diff
@@ -19,43 +37,57 @@ const ProfileToken = ({ token }) => {
...
+
+    setShouldSelectNewPFP(false)
+
+    await updateUserCre8orNumber({
+      walletAddress: address,
+      cre8orNumber: token?.tokenId,
+    })
+
+    await getUserData(address)
+  }
```
Comment (sweetmantech) [inline components/ProfilePage/ProfileToken.tsx]: good implementation 👏

File: providers/UserProvider.tsx [hunk 3]
```diff
@@ -53,6 +53,7 @@ export const UserProvider: FC<Props> = ({ children }) => {
   const [metaData, setMetaData] = useState<any>(null)
   const [similarProfiles, setSimilarProfiles] = useState<any>([])
   const { cre8orNumber, getCre8orNumber } = useCre8orNumber({ address: routerAddress || address })
+  const [dbCre8orNumber, setDBcre8orNumber] = useState("")
```
Comment (sweetmantech) [inline providers/UserProvider.tsx]: why are we creating a new `dbCre8orNumber` variable?

Shouldn't `cre8orNumber` automatically update when you save the new valua to the DB?

**Review by SweetmanTech — COMMENTED** [review]
why are we adding `dbCre8orNumber`?

File: components/ProfilePage/OpenSeaNewPFP.tsx [hunk 4]
```diff
@@ -2,49 +2,59 @@
...
+        <Button
+          id="opensea_via_smart_wallet"
+          className="!w-[250px] !h-[50px]
+              md:!w-[513px] md:!h-[103px]
+              !p-0
+              !bg-black
+              !text-white !text-[12px] md:!text-[23px]"
+        >
+          LOG IN TO OPENSEA VIA SMART WALLET
+        </Button>
```
Comment (sweetmantech) [inline components/ProfilePage/OpenSeaNewPFP.tsx]: `disable` this until we've added the functionality please

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**